src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/NewUser.vue

```diff
@@ -106,38 +136,12 @@ export default defineComponent({
   data: () => ({
     hasAgreed: false,
     eceRegistrationNumber: "",
-    eceCertificateStatus: null as boolean | null,
-    hasAgreedRules: [(v: boolean) => !!v || "You must read and accept the Terms of Use"],
+    eceCertificateStatus: undefined as boolean | undefined,
     Rules,
   }),
 
-  computed: {
-    phoneRules() {
-      return [...this.customPhoneRule(), this.Rules.required()];
-    },
-    eceRegistrationRules() {
-      return [
-        (v: string) => {
-          if (this.eceCertificateStatus === true) {
-            return !!v || "Enter your ECE registration number"; // Required if "Yes" selected
-          }
-          return true; // No validation if "No" selected
-        },
-      ];
-    },
-    radioRules() {
-      return [
-        (v: boolean | null) => v !== null || "Choose an option", // Radio button required
-      ];
-    },
-  },
+  computed: {},
```

Comment:
we can remove computed if there's nothing in it.

Reply:
Removed!